internal/orchestration/controller.go

```diff
@@ -276,11 +278,11 @@ func (e *Experiment) assignReplicasAndClients() (err error) {
 			replicaOpts.ByzantineStrategy = byzantineStrategy
 			replicaLocations = append(replicaLocations, location)
 			// all replicaOpts share the same Locations slice, which is progressively updated
-			replicaOpts.Locations = replicaLocations
 			e.hostsToReplicas[host] = append(e.hostsToReplicas[host], nextReplicaID)
 			e.replicaOpts[nextReplicaID] = replicaOpts
 			e.Logger.Infof("replica %d assigned to host %s", nextReplicaID, host)
 			nextReplicaID++
+			totalReplicaCount++
```

Comment:
You don't need this variable; you can just use `numReplicas`

Reply:
My bad. I see now `totalReplicaCount` is used outside the loop where `numReplicas` is used.